week-3/learning-outcomes.md

```diff
@@ -0,0 +1,26 @@
+# Week 3 Learning Outcomes
+
+## Testing
+
+- [ ] **We can test the functionality of our app**
+- [ ] We can write tests to mimic the behaviour of a user performing different actions
+- [ ] We can check that passing a given input into our tests returns the expected output
+
+### Modular functions
+
+- [ ] **We can write testable, modular functions**
+
+### HTML forms
+
+- [ ] **We can submit an HTML form without using JavaScript**
```

Comment:
This is probably a better LO for week 1. This week they aren't going to have a server and will be making `fetch` requests in JS. My previous comment was just about using a "click" event rather than a "submit" event

```diff
- button.addEventListener("click", handle)
+ form.addEventListener("submit", handle)`
```

Reply:
ok, that makes sense. I'm thinking something like "We can apply event listeners to HTML form elements" then.